frontend/package.json

```diff
@@ -18,7 +18,7 @@
         "@bcgov-nr/nr-theme": "^1.8.2",
         "@carbon/icons-vue": "^10.72.1",
         "aws-amplify": "^5.3.6",
-        "axios": "0.26.1",
+        "axios": "^1.6.5",
```

Comment:
Hi Ian, just wondering since the axios version for frontend and code-gen need to be the same, maybe we don't want to add "^" in front of the version number?

Reply:
Yes, I was questioning myself about that previously as well but feel might be ok (it seems to be a version high enough for stable).
But, I will change that to be fixed at this version, to be more comfortable not having break in the future. Thanks for mentioning that.

Reply:
This is now at fixed version.